load.php

```diff
@@ -262,7 +282,16 @@ function perflab_load_active_and_valid_modules() {
 		require_once PERFLAB_PLUGIN_DIR_PATH . 'modules/' . $module . '/load.php';
 	}
 }
-perflab_load_active_and_valid_modules();
+
+/**
+ * Loads the modules once the activated plugins have loaded.
+ *
+ * @since n.e.x.t
+ */
+function perflab_plugins_loaded() {
```

Comment:
I don't see a reason for adding a new callback here, let's just call `perflab_load_active_and_valid_modules` directly on `plugins_loaded`. Also, I worry that relying only on loading modules on `plugins_loaded` could allow for some edge cases where a standalone plugin could load after the module loads and still cause a fatal. It would be better if the modules themselves would short circuit if the constant were already defined.